testdata/networking/iperf_nodeport_service.json

```diff
@@ -0,0 +1,67 @@
+{
+  "apiVersion": "v1",
+  "kind": "List",
+  "items": [
+    {
+      "kind": "Pod",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "iperf-server",
+        "labels": {
+          "name": "iperf-server"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "name": "iperf-server",
+            "image": "bmeng/iperf3",
```

Comment:
Are we still using images from personal account ?

Reply:
Oh, is there an alternative account or registry?

Reply:
@xiuwang Could you help push `bmeng/iperf3` image to our owned registry? thanks

Reply:
@rbbratta this has not been addressed.

Reply:
@rbbratta Seems like this was pushed ```https://quay.io/openshifttest/iperf3@sha256:440c59251338e9fcf0a00d822878862038d3b2e2403c67c940c7781297953614```